### ISSUE TITLE
Can't use except validator condition

### ISSUE THREAD

**Issue by glagola** [yiisoft/yii2#7846]
Hi!

I'm trying to exclude my validation rules from creation scenario, which name specified in ```ActiveController::createScenario```

### Example
```
public function rules() 
{
    return [
         ['name', 'required', 'except' => 'create'],
    ];
}
```

As you can see, I have no other rules for 'create' scenario, I'm using the scenario just to exclude validation rule.

But when I call ```Model::validate()``` it [throws "Unknown scenario" exception](https://github.com/yiisoft/yii2/blob/master/framework/base/Model.php#L324-L326).

Where is the bug? In [docs](http://www.yiiframework.com/doc-2.0/yii-validators-validator.html#$except-detail) or in the [code](https://github.com/yiisoft/yii2/blob/master/framework/base/Model.php#L219-L220)?

**Comment by cebe** [comment]
you could define scenarios yourself to work around this. Model should generate scenarios by also including information from except.

**Comment by glagola** [comment]
@cebe By the way, maybe it is also a bug: If rule has specified no $on, and no $except, it would not be common  for all scenarios

**Comment by klimov-paul** [comment]
Resolved by commit a4f760e
